modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -65,3 +67,24 @@
 	desc = "A gray naval suit with a lead-lined vest and a rank badge denoting the Officer of the Internal Engineering Division. Doesn't come with a death machine building guide."
 	name = "chief engineer's naval jumpsuit"
 	icon_state = "impce"
+
+/*
+*	ATMOS TECH
+*/
+/datum/armor/atmos_adv
+	melee = 30
+	bullet = 30
+	laser = 30
+	energy = 30
+	bomb = 30
+	fire = 70
+	acid = 70
+	wound = 20
+
+/obj/item/clothing/under/rank/engineering/atmospheric_technician/skyrat/utility/advanced
```

Comment:
why are you armouring their jumpsuit instead of their modsuit?

Reply:
as a note, we advise against armor on jumpsuits at /tg/ because of how armor stacking interacts

Reply:
Ah okay. I was pulling from some other jumpsuit. I'll adjust.